scripts/build/builders/bouffalolab.py

```diff
@@ -103,6 +104,8 @@ def __init__(self,
 
         if bouffalo_chip == "bl702":
             self.argsOpt.append('module_type=\"{}\"'.format(module_type))
+            if enable_cdc:
```

Comment:
should we raise if enable_cdc is set and chip is not bl702?

maybe targets.py should have a `onlyif` restriction